Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs

```diff
@@ -18,7 +19,7 @@ public class OpenVRDeviceManager : BaseDeviceManager
         /// <summary>
         /// Dictionary to capture all active controllers detected
         /// </summary>
-        private readonly Dictionary<XRNode, IMixedRealityController> activeControllers = new Dictionary<XRNode, IMixedRealityController>();
+        private readonly Dictionary<XRNode, GenericOpenVRController> activeControllers = new Dictionary<XRNode, GenericOpenVRController>();
```

Comment:
not the interface?

Reply:
In this case no, else we have to do lots of boxing operations.  They are all subclasses of the Generic controller

Reply:
On a side note, shouldn't this class be in _Core/Managers/ ?

Reply:
No, because it's specific to the device layer.